src/conn/conn_stat.c

```diff
@@ -76,6 +76,7 @@ __wt_conn_stat_init(WT_SESSION_IMPL *session)
     __wti_cache_stats_update(session);
     __wt_evict_stats_update(session);
     __wt_txn_stats_update(session);
+    __wt_checkpoint_set_time_stats(session, &conn->ckpt);
```

Comment:
Happy to rename and change the arguments of this function so it's more consistent with the ones used above. Please let me know what you prefer.

Reply:
sounds better, in the function itself we can get hold of ckpt.

Reply:
Sure, done in 8c90dae36e982095133070f31ba0ef94cb81640f and I have also updated another function for consistency, see 80d16453745dbe26a30840fefd92c23ba0cf3213.

Reply:
Should I do the same for `__wt_checkpoint_reset_handle_stats` and `__wt_checkpoint_reset_time_stats`? I could also align the names and put the action (reset/update) at the end of the function's name:
```
__wt_checkpoint_stats_handle_update
__wt_checkpoint_update_handle_stats
vs
__wt_checkpoint_reset_handle_stats
__wt_checkpoint_reset_time_stats
```